client/my-sites/people/main.jsx

```diff
@@ -113,7 +112,6 @@ class People extends React.Component {
 							site={ site }
 						/>
 					}
-					<PeopleNotices />
```

Comment:
Are you sure that `PeopleNotices` are completely inactive on this page? The `people/log-store` reacts to actions like `RECEIVE_USERS` and these are not yet completely gone.

Reply:
All remaining actions are covered by open PRs (see #51879 and #51882). The `<SiteUsersFetcher />` (the data component for users) which also uses the flux `fetchUsers` action will then be unused and can be removed as well. 

However, not all of these usages are used in the `people` section and therefore not necessarily relevant to `<PeopleNotices />`. As far as I can see there's no remaining flux action dispatched when `<PeopleNotice />` is mounted. Otherwise I may be missing something.

Reply:
> As far as I can see there's no remaining flux action dispatched when `<PeopleNotice />` is mounted.

Yes, that's exactly what I see, too 🙂 Just wanted to confirm.